app/kubemci/cmd/version.go

```diff
@@ -22,22 +22,25 @@ import (
 )
 
 var (
-	versionDescription = "Prints the version of this tool"
+	versionShortDescription = "Prints the version of this tool"
+	versionLongDescription  = `Prints the version of this tool
+
+	Release builds have versions of the form x.y.z.
+	version x.y.z+ indicates that the client has some changes over the x.y.z release.
+	`
 )
 
 const (
 	// String indicating the client version.
-	// x.y.z+ indicates that the client has some changes over the x.y.z release.
-	// Release builds should have versions of the form x.y.z.
 	// TODO(nikhiljindal): Add more useful information such as build date, git commit, etc similar to kubectl.
-	clientVersion = "0.0.0+"
+	clientVersion = "0.0.1"
```

Comment:
Can you include something here, like the git commit and date of creation for 0.0.1?

Reply:
You mean as a comment in code or in the output string visible to user?

fwiw, I will also create a release tag at https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/releases which will have all that information.

Reply:
kubectl has a more sophisticated versioning logic. This is very basic :)
 I have a TODO to fix this.
Leaving as is for now.